doc/api/path.md

```diff
@@ -36,7 +91,7 @@ process.env.PATH.split(path.delimiter)
 // returns ['/usr/bin', '/bin', '/usr/sbin', '/sbin', '/usr/local/bin']
 ```
 
-An example on Windows:
+And on Windows:
```

Comment:
Minor nit, but this should probably have either s/And/and/ *or* s/And// and s/on/On/.